infra/examples-dev/aws-msft-365/main.tf

```diff
@@ -71,13 +71,13 @@ module "worklytics_connector_specs" {
   # source = "git::https://github.com/worklytics/psoxy//infra/modules/worklytics-connector-specs?ref=v0.4.6"
 
   enabled_connectors = [
-    "asana",
-    "azure-ad",
+    // "asana",
+    // "azure-ad",
     "dropbox-business",
-    "outlook-cal",
-    "outlook-mail",
-    "slack-discovery-api",
-    "zoom"
+    // "outlook-cal",
+    // "outlook-mail",
+    // "slack-discovery-api",
+    //"zoom"
```

Comment:
personal situation ... leave uncommented generally. I have task to migrate to a variable.

Reply:
Yeah sorry, I've forgot to leave it uncommented